app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,57 @@
+class InvitesController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:confirm_request, :create_staff_user]
```

Comment:
why aren't we using load_and_authorize here

Reply:
Because outside user who has received invitation is going to use these functions to register.